### PR TITLE
chore(server): change transaction fee calculation way

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

File: server/app/models/ckb_sync/persist.rb [hunk 1]
```diff
@@ -225,7 +226,8 @@ def build_cell_input(ckb_transaction, input)
         ckb_transaction.cell_inputs.build(
           previous_output: input["previous_output"],
           since: input["since"],
-          args: input["args"]
+          args: input["args"],
+          from_cell_base: input["previous_output"]["cell"].present? ? false : true
```
Comment (classicalliu) [inline server/app/models/ckb_sync/persist.rb]:
```suggestion
          from_cell_base: !input["previous_output"]["cell"].present?
```

Comment (duanyytop) [inline server/app/models/ckb_sync/persist.rb]: `! input["previous_output"]["cell"].present?` maybe more better ?

Comment (shaojunda) [reply server/app/models/ckb_sync/persist.rb]: Thank you, I found it more appropriate to use `blank?`.

Comment (shaojunda) [reply server/app/models/ckb_sync/persist.rb]: Thank you.

**Pull request merged**